ui/src/app/cohort-review/detail-all-events/detail-all-events.component.css

```diff
@@ -14,8 +14,18 @@ clr-dg-cell.cell {
   max-height: calc(100vh - 260px);
   min-height: 20rem;
   overflow-y: auto;
+  overflow-x: auto;
 }
 
 .gray{
   background-color: #E2E2EA;
 }
+
+::ng-deep .datagrid-compact .datagrid-foot {
+  height: 1.5rem;
+  line-height: calc(0.4rem - 1px);
+}
+
+::ng-deep .datagrid-foot-description {
+  padding-top: 0.3rem;
+}
```

Comment:
I think we should use `:host::deep` here so it only traverses down the component tree instead of applying to the entire app. I think Scout brought this up in a recent meeting and I've need to update some of my style also

Reply:
Ok.. I will change it.